packages/edit-site/src/index.js

```diff
@@ -13,6 +13,7 @@ import { render } from '@wordpress/element';
  */
 import './hooks';
 import './store';
+import './plugins';
```

Comment:
I'm not sure it matters, but `edit-post` imports hooks, then plugins, then the store. We should probably mirror that just to be safe since these have side effects.

Reply:
Also removed the plugins file in the refactor so this is no longer present.